index.rst

```diff
@@ -154,6 +154,7 @@ Binary Sensor Components
     PN532, components/binary_sensor/pn532, pn532.jpg
     RDM6300, components/binary_sensor/rdm6300, rdm6300.jpg
     TTP229-LSF  Capacitive Touch Sensor, components/binary_sensor/ttp229, ttp229.jpg
+    TTP229-BSF  Capacitive Touch Sensor, components/binary_sensor/ttp229_bsf, ttp229_bsf.png
```

Comment:
Convert the image to a jpg please. JPG is much smaller than PNG for such a image (and we don't need transparency here)